code/game/objects/items/storage/pouch.dm

```diff
@@ -10,12 +10,15 @@
 	allow_drawing_method = TRUE
 	var/fill_type
 	var/fill_number = 0
+	///Defines how many versions of the empty sprite there are
+	var/sprite_slots = null
```

Comment:
This should probably be at the storage level, not pouch, especially since you already have said you plan to do this for other storage types.

Reply:
Gotcha. There seems to be no existing update_icon_state() so it is in there now.
I was not sure where to put it and all of the codes in storage.dm are all procs and verbs so I pasted them in the last line. Line 869
![image](https://github.com/tgstation/TerraGov-Marine-Corps/assets/58515178/84c32671-6ff7-4616-a0d9-fa042e52c9e6)

Reply:
I added a check for if a storage is completely empty, and set the icon_state to have "_e" instead of "_0", since _0 representing a bag being empty is a too odd of a convention and I want to avoid confusion from whoever touches pouches and belts.